engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptionAttachAmountFilter.filter.js

```diff
@@ -8,8 +8,16 @@
 angular.module('Bastion.subscriptions').filter('subscriptionAttachAmountFilter',
     ['translate',
     function (translate) {
-        return function (subscription) {
-            var amount = subscription.amount;
+        var amount;
+        var i;
```

Comment:
Actually, do these need to live outside the function using them in the return statement below?